types/Errors.d.ts

```diff
@@ -46,6 +49,10 @@ declare module 'stripe' {
       source?: Stripe.Source;
     };
 
+    /**
+     * @deprecated The {@link Stripe.StripeError} type is deprecated, please use {@link Stripe.errors.StripeError}.}
```

Comment:
This and most other @deprecated messages have a trailing `}` - let's remove.

eg.

```
     * @deprecated The {@link Stripe.StripeError} type is deprecated, please use {@link Stripe.errors.StripeError}.}
```

should be

```
     * @deprecated The {@link Stripe.StripeError} type is deprecated, please use {@link Stripe.errors.StripeError}.
```